src/main/resources/data/spookytime/loot_tables/entities/pumpcown.json

```diff
@@ -0,0 +1,71 @@
+{
+  "type": "minecraft:entity",
```

Comment:
Consider using in-code loot table generator once #7 is merged

Reply:
Or consider using the system vanilla put into place ages ago, for a good reason.